Packs/OpenCTI/ReleaseNotes/1_0_11.md

```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### OpenCTI
+
+- Updated the Docker image to: *demisto/vendors-sdk:1.0.0.92984*.
+- filters object in the get_indicators function changed according to the new version of OpenCTI
```

Comment:
Too much details for release notes. Main point should be "Switched x command to work with version x.y of opencti)